README.md

```diff
@@ -1,33 +1,11 @@
-# Operate First template for repositories
+# Operate First - OpenShift JSON schemas
 
-Derive new repositories from this template
+This repository stores OpenShift JSON schemas used with Kubeval tool to validate YAML Kubernetes configuration files.
 
-List of featurese:
+## Structure of the repository
 
-## License
+JSON files are stored in `schemas/master-standalone-strict`.  Master name is used in Kubeval to define default version of Kubernetes and standalone is used for schemas, that are de-referenced (meaning without URL references) and can be used as standalone documents. Strict is used for schemas which do no allow arbitrary attributes.
```

Comment:
Find this section a bit confusing, seems like you want to explain the reasoning behind the way you named the directory. I would instead reword it like this: I would state it something like: 

---
 Schemas are stored as JSON files in `schemas/` at the root of the repo. The directories within `schemas` are named using the following format: `${k8s-version}-${mode}-${strictvalue}`, where: 
```
k8s-version = bla bla 
mode = bla bla 
strictvalue = bla bla  
```

I also don't fully understand what you are trying to say here:

> that are de-referenced (meaning without URL references)

In other words not sure what standalone means.

Reply:
Thank you for an idea :+1: . I rewrote the README so it is hopefully little bit more helpful.